atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -113,13 +115,28 @@ def __init__(self, tasker, workflow, koji_hub,
     def is_image_build_type(self, base_image):
         return base_image.strip().lower() == 'koji/image-build'
 
+    def extract_base_url(self, repo_url):
+        response = requests.get(repo_url)
+        response.raise_for_status()
+        repo = ConfigParser()
```

Comment:
Should this be RawConfigParser or perhaps SafeConfigParser? Or maybe there's a reason to use ConfigParser here, but the docs say to prefer the Safe variant.